yt_dlp/extractor/pornhub.py

```diff
@@ -77,8 +77,8 @@ def _login(self, host):
         # on both sites so that we have to identify them as the same video.
         # For that purpose we have to keep both in the same extractor
         # but under different netrc machines.
-        username, password = self._get_login_info(netrc_machine=site)
-        if username is None:
+        email, password = self._get_login_info(netrc_machine=site)
+        if email is None:
```

Comment:
I would only change the form field
```suggestion
        username, password = self._get_login_info(netrc_machine=site)
        if username is None:
```

Reply:
I set the internal variables to `email` so that they would be more useful in verbose debugging

Reply:
these lines won't show up in the traceback though

Reply:
OK, will revert